tracer/src/Datadog.Trace/TracerManager.cs

```diff
@@ -203,6 +204,9 @@ public static void UnsafeReplaceGlobalManager(TracerManager instance)
         /// </summary>
         internal void Start()
         {
+            // Start the Serverless Mini Agent in GCP Functions & Azure Consumption Plan Functions.
+            ServerlessMiniAgent.StartServerlessMiniAgent(this.Settings);
```

Comment:
[coding style] We don't generally use `this` in this repo. (I thought the editorconfig settings would catch this.)

```suggestion
            ServerlessMiniAgent.StartServerlessMiniAgent(Settings);
```